deployment/terraform/container_service.tf

```diff
@@ -0,0 +1,201 @@
+locals {
+  app_image = "${module.ecr-repository.repository_url}:${var.image_tag}"
+}
+
+#
+# Security Group Resources
+#
+resource "aws_security_group" "alb" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgLoadBalancer${title(var.project)}${title(var.environment)}"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_security_group" "ecs_cluster" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgEcsCluster${title(var.project)}${title(var.environment)}"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+#
+# ALB Resources
+#
+resource "aws_lb" "oar" {
+  name            = "alb${title(var.project)}${title(var.environment)}"
```

Comment:
We typically do not include the project name again in the resource name. In this case, everything in the account will be associated with the OAR project.